packages/lib/src/vis/tiles/TiledHeatmapMesh.tsx

```diff
@@ -74,25 +76,32 @@ function TiledHeatmapMesh(props: Props) {
 
   const { colorMap, invertColorMap = false } = colorMapProps;
 
+  const tooltipLayer = layers[layers.length - 1];
+
   return (
-    <group ref={groupRef}>
-      <mesh position={[0, 0, -0.1]}>
-        <planeGeometry args={[meshSize.width, meshSize.height]} />
-        <meshBasicMaterial
-          color={getInterpolator(colorMap, invertColorMap)(0)}
-        />
-      </mesh>
-      {layers.map((layer) => (
-        <TiledLayer
-          key={layer}
-          api={api}
-          layer={layer}
-          meshSize={meshSize}
-          visibleBox={visibleBox}
-          {...colorMapProps}
-        />
-      ))}
-    </group>
+    <>
+      <group ref={groupRef}>
+        <mesh position={[0, 0, -0.1]}>
+          <planeGeometry args={[meshSize.width, meshSize.height]} />
+          <meshBasicMaterial
+            color={getInterpolator(colorMap, invertColorMap)(0)}
+          />
+        </mesh>
+        {layers.map((layer) => (
+          <TiledLayer
+            key={layer}
+            api={api}
+            layer={layer}
+            meshSize={meshSize}
+            visibleBox={visibleBox}
+            {...colorMapProps}
+          />
+        ))}
+      </group>
+      {showTooltip && tooltipLayer !== undefined && (
+        <TiledTooltipMesh layer={tooltipLayer} api={api} meshSize={meshSize} />
```

Comment:
For now the tooltip component is part of `TiledHeatmapMesh` but is relevant only for one layer that is given in prop.

We could imagine putting `TiledTooltipMesh` in the layer instead but then we should make sure to display only the tooltip of the above layer

Reply:
Not sure, as it is, you want a single tooltip at a time.
And to me, it sounds better in the end to have it outside of the `TiledHeatmapMesh` in order to have a single tooltip displayed over multiple meshes, but not for this PR.